packages/shared/styles/components/organisms/SfProductCard.scss

```diff
@@ -37,6 +37,23 @@
     text-decoration: none;
     margin: var(--product-card-link-margin, 0);
   }
+  &__colors {
+    --color-picker-label-font-size: var(--font-size--base);
```

Comment:
Would it be possible to somehow get rid of these CSS variables?

Reply:
this will be done in separate PR